src/js/api/urls.js

```diff
@@ -149,3 +149,9 @@ export const PRODUCT_SUPPLIER_ATTRIBUTE_BATCH = `${PRODUCT_SUPPLIER_ATTRIBUTE_AP
 
 export const PICKLIST_API = `${API}/picklists`;
 export const PICKLIST_CLEAR = id => `${PICKLIST_API}/${id}/items`;
+
+// FULL OUTBOUND IMPORT FEATURE
+export const FULFILLMENT_API = `${API}/fulfillments`;
```

Comment:
As I mentioned in the UrlMappings, since the Fulfillments API hasn't been fleshed out as well as I'd like let's make this a little more straightforward by using the PackingListController for everything.